zio-http/src/main/scala/zhttp/http/Cookie.scala

```diff
@@ -125,6 +142,34 @@ final case class Cookie(
     cookie.flatten.mkString("; ")
   }
 
+  /**
+   * Signs cookie content with a secret
+   */
+  def sign(secret: String, content: String): String = {
+    try {
```

Comment:
remote try

Reply:
`getInstance` and `doFinal` throws exceptions such as `NoSuchAlgorithmException` or `IllegalStateException`

Reply:
Is there a way we can avoid it? Like guarantee that the algorithm exists?